BluetoothExplorer/Controller/GATT Characteristic Editors/DateTimeCharacteristicViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  DateTimeCharacteristicViewController.swift
+//  BluetoothExplorer
+//
+//  Created by Carlos Duclos on 6/25/18.
+//  Copyright © 2018 PureSwift. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import CoreData
+import Bluetooth
+import GATT
+
+final class DateTimeCharacteristicViewController: UIViewController {
+    
+    // MARK: - IB Outlets
+    
+    @IBOutlet private(set) weak var dateTimeLabel: UILabel!
+    
+    @IBOutlet private(set) weak var datePicker: UIDatePicker!
+    
+    // MARK: - Properties
+    
+    var value = GATTDateTime(date: Date())
+    
+    var valueDidChange: ((GATTDateTime) -> ())?
+    
+    var minimumDate: Date? {
```

Comment:
Can this not be optional? Also can you make it a `private lazy var` so it is calculated only once.

Reply:
I can make it not optional if I use force unwrapping xD

Reply:
No, use guard with `fatalError` like in you are doing in `viewDidLoad`.